check-mdn-bcd.js

```diff
@@ -0,0 +1,28 @@
+const { writeFile } = require('fs')
+const { get } = require('https')
+
+const pkg = require('./package.json')
+
+get('https://registry.npmjs.org/@mdn/browser-compat-data', res => {
+  if (res.statusCode < 200 || res.statusCode >= 299) {
+    process.stderr.write(`${res.statusCode} response from npm\n`)
+    process.exit(1)
+  }
+
+  let data = ''
+  res.on('data', chunk => {
+    data += chunk
+  })
+  res.on('end', () => {
+    let body = JSON.parse(data)
+    let lastVersion = body['dist-tags'].latest
+    if (pkg.devDependencies['@mdn/browser-compat-data'] !== lastVersion) {
+      pkg.devDependencies['@mdn/browser-compat-data'] = lastVersion
+      writeFile('./package.json', `${JSON.stringify(pkg, null, 2)}\n`, () => {
```

Comment:
Only updates package.json currently presumably the workflow will update the lock when it runs install?

Reply:
It will be good to call `yarn install` to test new data with fresh changes

Reply:
My change to check.yml was intended to trigger yarn install when either caniuse or mdn has a new version. Is that not what you intended?

Reply:
You were right. We are calling `yarn install` after `check-mdn-bcd`.